installer.sh

```diff
@@ -419,6 +421,20 @@ mkdir -p /etc/keylime
 mkdir -p config
 python3 -m keylime.cmd.convert_config --defaults --out config --templates templates
 
+# set the default binding and listening IP about the agent, verifier, and registrar server as 0.0.0.0 when LISTEN_ALL_IPS is set 1
+for arg in "$@"
+do
+    if [[ $arg == LISTEN_ALL_IPS=* ]]; then
```

Comment:
IIUC the parameter will be used like this?

```
./install.sh LISTEN_ALL_IPS=1
```

If so that is not how it is expected. There is code to parse the parameters here: https://github.com/keylime/keylime/blob/a19bac371a8d38858882fd4e288aa61aa1f0a141/installer.sh#L187